.dockerignore

```diff
@@ -0,0 +1,11 @@
+.git/
+.gitattributes
+.gitignore
+.travis.yml
+changelog.md
+LICENSE
+README*
+THIRDPARTY
+Vagrantfile
+frontend_tests/
+docker-compose*.yml
```

Comment:
If not, can you add a comment at the top of this explaining how one should decide which things need to go in here?